packages/playwright-core/src/server/types.ts

```diff
@@ -373,3 +374,8 @@ export type AndroidDeviceOptions = {
   port?: number,
   omitDriverInstall?: boolean,
 };
+
+export type ServiceWorkerPolicy = {
```

Comment:
I'd suggest an enum with 3 values, something like default/disabled/inspected.

Reply:
Updating—this was an artifact of an evolving policy where there were more options.